torch/csrc/utils/linalg_backends.cpp

```diff
@@ -0,0 +1,40 @@
+#include <torch/csrc/utils/linalg_backends.h>
+
+#include <ATen/LinalgBackend.h>
+
+#include <torch/csrc/DynamicTypes.h>
+#include <torch/csrc/Exceptions.h>
+#include <torch/csrc/LinalgBackend.h>
+
+#include <torch/csrc/python_headers.h>
+#include <torch/csrc/utils/object_ptr.h>
+
+
+namespace torch {
+namespace utils {
+
+#define _ADD_LINALG_BACKEND(format)                                             \
+  {                                                                             \
+    std::string name = at::LinalgBackendToString(format);                       \
+    PyObject* linalg_backend = THPLinalgBackend_New(format);                    \
+    Py_INCREF(linalg_backend);                                                  \
+    if (PyModule_AddObject(torch_module, name.c_str(), linalg_backend) != 0) {  \
+      throw python_error();                                                     \
+    }                                                                           \
+  }
+
+void initializeLinalgBackends() {
+  auto torch_module = THPObjectPtr(PyImport_ImportModule("torch"));
```

Comment:
Why not put this in `torch.backends.linalg`? It looks surprising to have these in `torch.`